src/httpserver.cpp

```diff
@@ -141,13 +140,6 @@ class WorkQueue
         running = false;
         cond.notify_all();
     }
-    /** Wait for worker threads to exit */
-    void WaitExit()
-    {
-        std::unique_lock<std::mutex> lock(cs);
-        while (numThreads > 0)
```

Comment:
In commit "http: Remove WaitExit from WorkQueue"

It looks like the numThreads variable is unused now, so it might be worth removing as well.

Reply:
Thanks, I removed those in a new commit.